types/vcf/index.d.ts

```diff
@@ -0,0 +1,153 @@
+// Type definitions for vcf 2.0
+// Project: https://github.com/jhermsmeier/node-vcf
+// Definitions by: mcpar-land <https://github.com/mcpar-land>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.1
+
+/// <reference types="node" />
+
+// tslint:disable-next-line npm-naming
+export default vCard;
```

Comment:
```suggestion
export = vCard;
```

When you [require the package](https://runkit.com/embed/z2ouwvko24md) it does not have a `default`.